config/gtasa/constants.xml

```diff
@@ -523,67 +552,1539 @@
       <Constant Name="WEAPONTYPE_UNIDENTIFIED"/>
       <Constant Name="WEAPONTYPE_ANYMELEE"/>
       <Constant Name="WEAPONTYPE_ANYWEAPON"/>
-      <Constant Name="WEAPONTYPE_FLARE"/>
     </Enum>
-
-    <!-- Unknown Enum Constants -->
     <Enum Name="GANG">
-      <!-- 0, 1, 2, 3, 7 (?) used by the game scm -->
+      <Constant Name="GANG_FLAT"/>
+      <Constant Name="GANG_GROVE"/>
+      <Constant Name="GANG_NMEX"/>
+      <Constant Name="GANG_SFMEX"/>
+      <Constant Name="GANG_VIET"/>
+      <Constant Name="GANG_MAFIA"/>
+      <Constant Name="GANG_TRIAD"/>
+      <Constant Name="GANG_SMEX"/>
+      <Constant Name="GANG9"/>
+      <Constant Name="GANG10"/>
     </Enum>
     <Enum Name="RADIO">
-      <!-- 0, 1, 2, 3, 5, 6, 7, 10, 11 used by the game scm -->
+      <Constant Name="RS_CLASSIC_HIP_HOP"/>
+      <Constant Name="RS_COUNTRY"/>
+      <Constant Name="RS_CLASSIC_ROCK"/>
+      <Constant Name="RS_DISCO_FUNK_SOUL"/>
+      <Constant Name="RS_HOUSE_CLASSICS"/>
+      <Constant Name="RS_MODERN_HIP_HOP"/>
+      <Constant Name="RS_MODERN_ROCK"/>
+      <Constant Name="RS_NEW_JACK_SWING"/>
+      <Constant Name="RS_REGGAE"/>
+      <Constant Name="RS_RARE_GROOVE"/>
+      <Constant Name="RS_TALK"/>
+      <Constant Name="RS_OFF"/>
     </Enum>
     <Enum Name="VIS">
```

Comment:
Interesting, they didn't seem to have used an enum in San Andreas for `SET_AREA_VISIBLE` and similar.

Reply:
Yes, they used the plain ints for SET_AREA_VISIBLE, so I think this enumeration should be removed. I'm assuming it's mostly an interior id?

Reply:
Yeah, as the virtual world system was vastly different from Vice I think it'd make as much sense to enumerate them with string constants. Removed in https://github.com/thelink2012/gta3sc/pull/133/commits/abb49d2b87c54c3dae0fa859a45d0f7fbf74818b